qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -2451,10 +2453,6 @@ int FitPropertyBrowser::getAllowedIndex(int currentIndex) const {
     return 0;
   }
 
-  if (currentIndex == m_oldWorkspaceIndex) {
```

Comment:
I have no idea what this check was doing or why it was here :thinking:

Reply:
Me either perhaps it was to catch an invalid index (i.e. <0) which used to occur when this condition was met? Removing it doesn't seem to have done any harm...